apps/charterafrica/src/payload/utils/constants.js

```diff
@@ -0,0 +1,4 @@
+export const ORGANIZATION_COLLECTION = "organisations";
```

Comment:
I'm not a fun of functional-based grouping (`utis/constant`s, `utils/functions`, etc.). I prefer feature-based e.g. `utils/collections`, etc.